docker-nginx/logrotate.conf

```diff
@@ -0,0 +1,12 @@
+/var/log/nginx/*log {
+	daily
+	su root adm
+	rotate 4
+	missingok
+	notifempty
+	compress
+	sharedscripts
+	postrotate
+		kill -USR1 `cat /var/run/nginx.pid` &>/dev/null
```

Comment:
does this cause the container to restart itself?

Reply:
https://nginx.org/en/docs/control.html

It doesn't kill the app. It instead causes nginx to open a new file for logging

Reply:
Thanks for the clarification. This looks great